### DISCUSSION TITLE
DAI 2024-2025 - Practical work 2 - Tristan Baud, Mathieu Emery

### DISCUSSION THREAD

**Discussion by mathieuemery** [heig-vd-dai-course/heig-vd-dai-course#477] [Show and tell]
Un serveur de streaming où plusieurs clients peuvent se connecter pour recevoir un flux vidéo. Le client peut voir le contenu sans avoir besoin de télécharger la vidéo.

Lien de notre repo: [DAI-LAB-03](https://github.com/HEIG-VD-S3-PW/DAI-LAB-03)
Lien du release commit: [1c6aa1a](https://github.com/HEIG-VD-S3-PW/DAI-LAB-03/commit/1c6aa1ac7a693d7d464cc830562e4a39d547fec4)

**Comment by arnoheigvd** [comment]
Après discussion avec @mathieuemery et Tristan Baud, je rejoins ce groupe pour ce projet
